TECH_DEBT.md

```diff
@@ -0,0 +1,10 @@
+# Technical debt to pay back
+
+* We added a new custom Logger to service-front, to enable us to log as JSON and log MVC error events in more detail. Consequently, we should remove opg-lpa-logger from service-front/composer.json, as it's no longer used. However, removing this and updating composer.lock causes the app to break. Need to investigate why that is and upgrade composer packages.
```

Comment:
Probably a daft question, but were you manually doing this, or did you try removing it via composer cli?